src/main/java/com/boot/user/client/CartServiceClient.java

```diff
@@ -0,0 +1,13 @@
+package com.boot.user.client;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+
+@FeignClient(name = "cart-service/cart")
```

Comment:
I don't understand the /cart part. is it working this way? we usually put there only the service name. and the request path should be '/'

Reply:
yeah, i works only with "cart-service/cart"